server/src/apiClient/SpoonacularClient.js

```diff
@@ -0,0 +1,30 @@
+import got from "got"
+import dotenv from "dotenv"
+dotenv.config()
+const key = process.env.API_KEY
+
+class SpoonacularClient {
+  static async searchRecipes(searchQuery) {
+    const url = `https://api.spoonacular.com/recipes/complexSearch/?apiKey=${key}&query=${searchQuery}`
+    try {
+      const apiResponse = await got(url)
+      const responseBody = apiResponse.body
+      return responseBody
+    } catch(err) {
+      return { err }
+    }
+  }
+
+  static async searchRecipePage(searchQuery, targetPage) {
+    const url = `https://api.spoonacular.com/recipes/complexSearch/?apiKey=${key}&query=${searchQuery}&offset=${targetPage*10}`
+    try {
+      const apiResponse = await got(url)
+      const responseBody = apiResponse.body
+      return responseBody
+    } catch(err) {
+      return { err }
+    }
+  }
+}
```

Comment:
nice work! SUPER clean. I do think the JSON.parse can be moved from the router to this class. However, I can see the argument to not have JSON.parse in here. your choice